src/main/driver_unified.cpp

```diff
@@ -130,15 +130,15 @@ int runCvc5(int argc, char* argv[], std::unique_ptr<cvc5::Solver>& solver)
   {
     if( inputFromStdin ) {
       // We can't do any fancy detection on stdin
-      solver->setOption("input-language", "smt2");
```

Comment:
Should this really use setOptionInternal if `input-language` is a common option?

Reply:
The point is to distinguish options that the user set vs. those that are set automatically.

In my opinion, it is cleaner to consider *all* options setting that main does in the same way, regardless of the category of the options. This was `fromUser` in `SolverEngine::setOption` really means "the user intentionally set this option".